### DISCUSSION TITLE
Dispersion wavelength Dependency

### DISCUSSION THREAD

**Discussion by devigili** [Telecominfraproject/oopt-gnpy#414] [Q&A] [answered]
Dear contributors,

I am writing with regard to the Dispersion wavelength dependency. 
At the moment I am simulating hundreds of channels (417) in the S+C+L bands along 4x80 km spans. In order to take into account the dispersion wavelength dependency I have tried to use the parameter `dispersion_slope` inserting in the equipment 
`"dispersion_slope": 90` s/m^3 ( obtained from [a] considering a G.652 D fiber: S0max < 0.092 ps/nm^2 km).

Unfortunately, eventhough I am extremizing the simulations increasing the input power to 5.00 dBm, I keep obtaining the same values of the channels **SNR NLI (signal bw, dB)** wheter inserting the dispersion_slope parameter or not, 

In other words, it seems like that in GNPy's opinion the dispersion wavelength dependency phenomenon is negligible even while simulating multiband systems.

Thanks in advance

[a] https://www.itu.int/rec/T-REC-G.652-201611-I/en

**Comment by ojnas** [comment]
I'm not sure whether dispersion slope should have an impact on NLI but I think it may depend on the model used (GN vs GGN). @AndreaDAmico, any comment?

However, from looking at the code it seems that a `dispersion_slope` parameter specified in the equipment file is even completely ignored (since there is no default value in json_io.py), which clearly looks like a bug. @jktjkt, I'm not sure how we should address this but we probably need to include default values for all fiber types in the equipment file.

**Reply by AndreaDAmico** [reply]
Yes, the dispersion slope should have an impact on the NLI generation, but I am not sure about the intensity of this effect. In the current version this is considered only in the GGN.

**Comment by EstherLerouzic** [comment, marked as answer]
Chromatic dispersion, effective area, Raman Gain coefficient, and nonlinear coefficient can now be defined with a scaling
along frequency in v2.8 release.
